maplibre_gl/lib/src/controller.dart

```diff
@@ -8,7 +8,7 @@ typedef OnMapClickCallback = void Function(
     Point<double> point, LatLng coordinates);
 
 typedef OnFeatureInteractionCallback = void Function(
-    dynamic id, Point<double> point, LatLng coordinates);
+    dynamic id, Point<double> point, LatLng coordinates, String layerId);
```

Comment:
My issue with this is that it changes the callback function signature. Fine since the library is not stable yet. But maybe it's type to make a `InteractionEvent` class to package all the attributes inside?

Reply:
Why not, but I don't think it's necessary, given that functionality is vital.